apps/jobs/views.py

```diff
@@ -539,3 +540,38 @@ def users(self, request):
         return response.create_response(
             serialized_company_data.data, status.HTTP_200_OK
         )
+
+
+class ContactUsViewSet(viewsets.ModelViewSet):
```

Comment:
people who submitted their queries via contact-us page, should only be visible to the moderator and not to general public. Handle such cases as well. 
Also, ensure that if GET request is made on this endpoint, then it should check for if the user who made the request is moderator or not, if `user` == `moderator` then only list the details of users who submitted their queries via contact-us page, or else just display nothing or disallow the request.

Reply:
As said, now the GET request will allow only moderator  i.e user == moderator to see the submitted queries. The image below defines the case where user =! moderator

![image](https://github.com/null-open-security-community/null-jobs-backend-2.0/assets/87690475/e4dc3e19-2ca6-468f-b2f1-6f7c6821bbe8)